lib/osm_community_index/local_chapter.rb

```diff
@@ -0,0 +1,64 @@
+module OsmCommunityIndex
+  class LocalChapter
+    attr_reader :id, :url
+
+    def initialize(id, url)
+      @id = id
+      @url = url
+    end
+
+    def self.local_chapters
+      @chapters = init_local_chapters
+    end
+
+    def self.init_local_chapters
+      raw_local_chapters = load_raw_local_chapters
+      local_chapters = []
+      raw_local_chapters.each do |chapter|
+        id = chapter[:id]
+        url = chapter[:resource]["strings"]["url"]
+        local_chapters.push(LocalChapter.new(id, url))
+      end
+      local_chapters
+    end
+
+    def self.load_raw_local_chapters
+      community_index = OsmCommunityIndex.community_index
+      raw_local_chapters = []
```

Comment:
`map ... .compact` rather than `foo = [] ... each ... unless x; foo.push(bar)` to make this easier to read. Or if we were on ruby 2.7+ (which we aren't yet) we could use [`Enumerable.filter_map`](https://blog.saeloun.com/2019/05/25/ruby-2-7-enumerable-filter-map.html)

Reply:
We are on 2.7+ as far as I know - it's what INSTALL says and it's what rubocop is targetting.

Reply:
Good point, I didn't actually check